packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -180,7 +181,16 @@ function getFrameworkTests(opts: {
 				flags: ["--typescript", "--no-install", "--no-git-init"],
 			},
 			next: {
-				testCommitMessage: false,
+				testCommitMessage: true,
+				flags: [
+					"--ts",
+					"--tailwind",
+					"--eslint",
+					"--app",
+					"--import-alias",
+					"@/*",
+					"--src-dir",
+				],
```

Comment:
Without these the create-next-app CLI hangs waiting for a response.

Reply:
At least locally. Perhaps it doesn't ask when in non-interactive mode?

Reply:
Add that as a comment?

Reply:
Also consider using `--yes` https://nextjs.org/docs/app/api-reference/cli/create-next-app#reference

Reply:
will try it out in a follow up